src/legacy/core_plugins/apm_oss/index.js

```diff
@@ -38,7 +38,7 @@ export default function apmOss(kibana) {
         spanIndices: Joi.string().default('apm-*'),
         metricsIndices: Joi.string().default('apm-*'),
         onboardingIndices: Joi.string().default('apm-*'),
-        cmIndex: Joi.string().default('.apm-cm')
+        cmIndex: Joi.string().default('.apm-ac')
```

Comment:
changed the default index name while i was here, to match the feature name. I'll change the variable names to match in another PR.

Reply:
I don't think anyone outside APM UI will know that "ac" means agent configuration - and I'll probably forget in a couple of weeks :D Perhaps we should avoid the abbreviation and write it out in full: `.apm-agent-configuration`?

Reply:
I also still see the abbreviation CM used in many places: https://github.com/elastic/kibana/search?q=path%3Ax-pack%2Flegacy%2Fplugins%2Fapm+CM&unscoped_q=path%3Ax-pack%2Flegacy%2Fplugins%2Fapm+CM

Reply:
Btw. shouldn't `cmIndex` also be renamed? Eg. `agentConfigurationIndex` - and exposed below?

Reply:
This PR was just a cherry-pick to get master back in sync with 7.x, since this was commited directly to 7.x to fix the breaking CI tests. Should we keep the scope of this PR down to this, or should we address these renaming issues in the other ticket along with other feedback?